### DISCUSSION TITLE
Fulfillment Policy - cap on max no of attempts for delivery

### DISCUSSION THREAD

**Discussion by BLR-0118** [beckn/protocol-specifications#289] [Core Specification]
Need an attribute in the fulfillment policy to define the cap on maximum number of attempts for delivery so the delivery agent knows the max no of times they should attempt delivery. This should be set by the logistics provider prior to confirmation of their order.

**Comment by Jaikant** [comment]
This could be part of the db schema, rather than the protocol?

**Comment by ravi-prakash-v** [comment]
@BLR-0118 why would this attribute be useful to a BAP user? Isn't this an internal policy of the provider?

**Comment by BLR-0118** [comment]
This should be published by the provider in its fulfillment policy so that anyone searching for their services could also use this as a criteria for filter

**Comment by Jaikant** [comment]
@BLR-0118 we shouldn't design the protocol for failure rate, especially human. Because the human is a totally flawed creature:)
